server/src/main/java/com/objectcomputing/checkins/services/agenda_item/AgendaItemServicesImpl.java

```diff
@@ -0,0 +1,90 @@
+package com.objectcomputing.checkins.services.agenda_item;
+
+import com.objectcomputing.checkins.services.checkins.CheckInRepository;
+import com.objectcomputing.checkins.services.memberprofile.MemberProfileRepository;
+
+import javax.inject.Inject;
+import javax.validation.constraints.NotNull;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+public class AgendaItemServicesImpl implements AgendaItemServices {
+
+    @Inject
+    private CheckInRepository checkinRepo;
+    @Inject
+    private AgendaItemRepository agendaItemRepo;
+    @Inject
+    private MemberProfileRepository memberRepo;
+
+    public AgendaItem save(AgendaItem agendaItem) {
+        AgendaItem agendaItemRet = null;
+        if (agendaItem != null) {
+            final UUID guildId = agendaItem.getCheckinid();
```

Comment:
Should this variable be named guildId? Maybe checkinId?

Reply:
Good catch

Reply:
Yes, updated the variable, thanks